### PR TITLE
change test such that it passes in headless mode (instead of xpass)

### LINKED ISSUE #48483: BUG: [unit test] pandas/io/clipboard/__init__.py:300: PyperclipException
### Pandas version checks

- [X] I have checked that this issue has not already been reported.

- [X] I have confirmed this bug exists on the [latest version](https://pandas.pydata.org/docs/whatsnew/index.html) of pandas.

- [X] I have confirmed this bug exists on the main branch of pandas.

### Reproducible Example

```python
# pytest pandas/tests/io/
============================================================================================ short test summary info =============================================================================================
FAILED pandas/tests/io/test_clipboard.py::TestClipboard::test_raw_roundtrip[\U0001f44d...] - pandas.errors.PyperclipException: 
FAILED pandas/tests/io/test_clipboard.py::TestClipboard::test_raw_roundtrip[\u03a9\u0153\u2211\xb4...] - pandas.errors.PyperclipException: 
FAILED pandas/tests/io/test_clipboard.py::TestClipboard::test_raw_roundtrip[abcd...] - pandas.errors.PyperclipException: 
========================================================================================= 3 failed, 232 passed in 1.55s ==========================================================================================
```

### Issue Description

The unit-test  https://github.com/pandas-dev/pandas/blob/047c11d7801ffd0cf679d606293af510e34d7b92/pandas/tests/io/test_clipboard.py#L398
does not pass.

I am using VSCode with the .devcontainer file (thus in a container), with a linux distribution (pop-os). 
Current commit I am on is 047c11d7801ffd0cf679d606293af510e34d7b92

I tried to do `apt-get install xclip` but it did not solve the issue. 

### Expected Behavior

The test should pass.

### Installed Versions

<details>

INSTALLED VERSIONS
------------------
commit           : 047c11d7801ffd0cf679d606293af510e34d7b92
python           : 3.8.13.final.0
python-bits      : 64
OS               : Linux
OS-release       : 5.8.0-7630-generic
Version          : #32~1609193707~20.10~781bb80-Ubuntu SMP Tue Jan 5 21:29:56 UTC 2
machine          : x86_64
processor        : x86_64
byteorder        : little
LC_ALL           : C.UTF-8
LANG             : C.UTF-8
LOCALE           : en_US.UTF-8

pandas           : 1.6.0.dev0+89.g047c11d780.dirty
numpy            : 1.22.4
pytz             : 2022.2.1
dateutil         : 2.8.2
setuptools       : 65.3.0
pip              : 22.2.2
Cython           : 0.29.32
pytest           : 7.1.3
hypothesis       : 6.54.5
sphinx           : 4.5.0
blosc            : None
feather          : None
xlsxwriter       : 3.0.3
lxml.etree       : 4.9.1
html5lib         : 1.1
pymysql          : 1.0.2
psycopg2         : 2.9.3
jinja2           : 3.0.3
IPython          : 8.5.0
pandas_datareader: 0.10.0
bs4              : 4.11.1
bottleneck       : 1.3.5
brotli           : 
fastparquet      : 0.8.3
fsspec           : 2021.11.0
gcsfs            : 2021.11.0
matplotlib       : 3.5.3
numba            : 0.55.2
numexpr          : 2.8.3
odfpy            : None
openpyxl         : 3.0.10
pandas_gbq       : 0.17.8
pyarrow          : 9.0.0
pyreadstat       : 1.1.9
pyxlsb           : 1.0.9
s3fs             : 2021.11.0
scipy            : 1.9.1
snappy           : 
sqlalchemy       : 1.4.41
tables           : 3.7.0
tabulate         : 0.8.10
xarray           : 2022.6.0
xlrd             : 2.0.1
xlwt             : 1.3.0
zstandard        : 0.18.0
tzdata           : None

</details>

### PR DESCRIPTION
**Pull request by adrienpacifico**
…and xpass in non-headless mode (instead of error).

- [ ] closes #48483 
- [ ] [Tests *modified* and passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#writing-tests). 
- [ ] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).
- [ ] Added [type annotations](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#type-hints) to new arguments/methods/functions.
- [ ] Added an entry in the latest `doc/source/whatsnew/vX.X.X.rst` file if fixing a bug or adding a new feature.

### CODE REVIEW

File: pandas/tests/io/test_clipboard.py [hunk 1]
```diff
@@ -391,9 +392,9 @@ def test_round_trip_valid_encodings(self, enc, df):
 
     @pytest.mark.parametrize("data", ["\U0001f44d...", "Ωœ∑´...", "abcd..."])
     @pytest.mark.xfail(
-        reason="Flaky test in multi-process CI environment: GH 44584",
```
Comment (mroeschke) [inline pandas/tests/io/test_clipboard.py:394]: Could you add the `@pytest.mark.single_cpu` marker on this test to replace this as well?

Comment (adrienpacifico) [reply pandas/tests/io/test_clipboard.py:394]: Like this ? https://github.com/pandas-dev/pandas/pull/48520/commits/932dad55d66db2a523ed109eb1ae880d44bffe60

Is there an easy way to squash my commits such that there is only one left for the merge?

Comment (mroeschke) [reply pandas/tests/io/test_clipboard.py:394]: Sure (thought typically we put these marks as the first decorator)

> Is there an easy way to squash my commits such that there is only one left for the merge?

No need to worry about it.  We squash all commits on merge.

Comment (adrienpacifico) [reply pandas/tests/io/test_clipboard.py:394]: Thank you.

> (thought typically we put these marks as the first decorator)

Oh! Sorry! 
It should be good now : https://github.com/pandas-dev/pandas/pull/48520/commits/25b5527934f04d5e97341b662e4bc9d71c615424

**Review by mroeschke — APPROVED** [review]

**Comment by mroeschke** [comment]
Great! Thanks @adrienpacifico

**Comment by jbrockmendel** [comment]
These are now xpassing for me on mac, py39

**Comment by adrienpacifico** [comment]
[jbrockmendel](https://github.com/jbrockmendel) and it was passing before the PR or was it xpassing? 
Sorry for the inconvenience. I guess the `strict = True` is breaking your test suite?

**Comment by jbrockmendel** [comment]
> I guess the strict = True is breaking your test suite?

correct

**Pull request merged**